tests/integration/test_dedup_backup.py

```
@@ -91,6 +93,18 @@ def test_multiple_backups_of_same_file(self, backup_mgr: BackupManager, tmp_path
         assert b2.exists()
 
+    def test_create_backup_missing_source_raises(
+        self, backup_mgr: BackupManager, tmp_path: Path
+    ) -> None:
+        with pytest.raises(FileNotFoundError):
+            backup_mgr.create_backup(tmp_path / "missing.txt")
+
+    def test_create_backup_directory_path_raises(
+        self, backup_mgr: BackupManager, tmp_path: Path
+    ) -> None:
+        with pytest.raises(ValueError):
```

Ruff (0.15.10)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)